src/app/app.component.ts

```diff
@@ -7,6 +7,7 @@ import { User } from './_models';
 @Component({ selector: 'app', templateUrl: 'app.component.html' })
 export class AppComponent {
     currentUser: User;
+    homeUrl = 'http://wwwapps.biz/';
```

Comment:
where are we using this homeUrl variable?

Reply:
homeUrl is used in footer